### ISSUE TITLE
when using pymc3 progress bar doesn't overwrite itself 

### ISSUE THREAD

**Issue by paulathomas** [jupyterlab/jupyterlab#6497]
Bug Description

In the previous version of jupyter lab the progress bar presented by the pymc3 module behaved as a normal slider. It now prints 100 times once for each percentage point. I have attached a screenshot of the current behaviour.
![Screenshot_2019-06-06_19-19-18](https://user-images.githubusercontent.com/3585330/59056930-2aba5200-8891-11e9-8189-29bb203abb2f.png)

**Comment by jasongrout** [comment]
What version of JupyterLab? Can you try with the latest prerelease, 1.0.0a6?

**Comment by paulathomas** [comment]
Sorry, should have said - it is 1.0.0a6. I had installed it from GitHub to
solve another issue. I have now gone back. This works on 0.35.6 and
previous versions.

**Comment by paulathomas** [comment]
I see this has status 'needs info'. Is there any more information required from me?

**Comment by jasongrout** [comment]
I think you gave us the info we needed. Thanks! After #6513 is released, it would be great if you could test if it fixed the issue. Or if you want to try a dev installation out with #6513, that would be great too.

**Comment by ian-r-rose** [comment]
I have verified that #6513 fixes this issue.

**Comment by paulathomas** [comment]
I will certainly test it when #6513 is released.
